### DISCUSSION TITLE
设置分享：DDNSTO 远程访问控制面板的方法

### DISCUSSION THREAD

**Discussion by vernesong** [vernesong/OpenClash#3745] [Show and tell]
## 插件设置
![image](https://github.com/vernesong/OpenClash/assets/42875168/8052553b-189b-40a1-a27d-c900875b2437)
## DDNSTO 映射设置
![image](https://github.com/vernesong/OpenClash/assets/42875168/0dfec596-ce98-4d3d-a89a-a6b440003e42)
## 设置完成后控制面板的访问地址
![image](https://github.com/vernesong/OpenClash/assets/42875168/72cdbd80-00ac-46c2-84c4-2a9308901551)
![image](https://github.com/vernesong/OpenClash/assets/42875168/2db09d3e-7e0f-4d70-b9c7-a9f4f54e2f17)
### 或者直接在插件主页点击控制面板按钮进行访问

**Comment by LuckWook** [comment]
感谢大佬指点

**Comment by tadi-huang** [comment]
还是不能访问啊，移动网络

**Reply by Choufish** [reply]
公网域名前面不要加“https://”

**Comment by Bennie-CHEN** [comment]
感谢

**Comment by feihuroger** [comment]
特意登录上来感谢。解决了大问题。

**Comment by rushidea** [comment]
完全照做，还是显示：{"message":"Unauthorized"} 😭

**Comment by vernesong** [comment]
0.46.090版本开始已经添加了跨域配置，现在按图设置应该没问题了

**Comment by EVANGELION-Air** [comment]
完全有效，非常感谢

**Comment by atna1988** [comment]
十分感谢，解决了大问题

**Comment by Jablani** [comment]
域名解析openwrt的IPv6地址，访问http://域名:9090/ui/yacd可以访问登录页，输入http://域名+端口和密码可以进入控制面板，访问http://域名:9090/ui/dashboard也能访问登录页，但不能登录。另外，用lucky做的反向代理可以访问yacd的登录页面，却不能登录。

**Reply by damaogou** [reply]
您好，我也遇到这种问题，但是我已经找到解决方法。使用lucky反向代理，api登录不要填写HTTPs，只填写域名加端口号，加密钥就行了。
